satellite/orders/service.go

```diff
@@ -191,6 +191,7 @@ func (service *Service) CreateGetOrderLimits(ctx context.Context, bucketID []byt
 	}
 
 	if len(limits) < redundancy.RequiredCount() {
+		mon.Meter("download_failed_not_enough_pieces_uplink").Mark(1) //locked
```

Comment:
Similar to Moby's comment below, we should log which uplink is getting this failure so we can initiate any necessary damage control with the customer.

Reply:
Do you think logging the bucketID would be sufficient enough?

Reply:
I added project id and apikey id into the log